docs/managing/connections.md

```diff
@@ -32,7 +32,9 @@ For example, the `aws` plugin will install the `~/.steampipe/config/aws.spc` con
 
 
 ### Syntax
-Steampipe config files use HCL Syntax, with connections defined in a `connection` block.  The `connection` name will be used as the Postgres schema name in the Steampipe database.  Each `connection` must contain a single `plugin` argument that specifies which plugin to use in this connection.  Additional arguments are plugin-specific, and are used to determine the scope, credentials, and other configuration items. 
+Steampipe config files use HCL Syntax, with connections defined in a `connection` block.  The `connection` name will be used as the Postgres schema name in the Steampipe database.  Each `connection` must contain a single `plugin` argument that specifies which plugin to use in this connection.  Additional arguments are plugin-specific, and are used to determine the scope, credentials, and other configuration items.
+
+Note: Connection names typically use lowercase characters and underscores. It's possible to use other characters, but be aware that the schema names derived from such connection names will need to be quoted in SQL. A statement like `select * from aws_profile_1.aws_account requires no quotation`. But a statement like `select * from "Aws:01(profile2)".aws_account` does require quotation.
```

Comment:
@kaidaguerre @e-gineer  I had thought that these needed to be valid [HCL identifiers](https://developer.hashicorp.com/terraform/language/syntax/configuration#identifiers) so I'm surprised that something like `Aws:01(profile2)` even parses...  I would propose that, at a minimum we enforce the hcl/terraform identifier rules (`Identifiers can contain letters, digits, underscores (_), and hyphens (-). The first character of an identifier must not be a digit, to avoid ambiguity with literal numbers.`).  And I wouldn't mind if we even enforced that they are also valid non-quoted postres identifiers (`Unquoted identifiers must begin with a letter or an underscore character. Subsequent characters can be letters, numbers, or underscores`) assuming we deprecated / warned first.  What do you guys think?

Reply:
I was unsure how they worked TBH. Makes sense to limit to valid identifiers, but the cat is already out of the bag a bit. We definitely see people use capital letters here, and that also leads to confusion.

Overall, I'm pro restricting these down to be valid, lowercase, postgres identifiers. But we'd need to go through a deprecation flow to get there.